### ISSUE TITLE
GPL, interested in changing to a BSD license?

### ISSUE THREAD

**Issue by jdfreder** [sebmatton/jQuery-Flight-Indicators#5]
Hey, great indicators you have here!  They are beautiful.  Did you pick the GPL on purpose, or was it a stab in the dark? Would you be willing to change to a BSD 3-clause so derivative projects could have the freedom to choose their own licensing?

**Comment by SylvainCorlay** [comment]
:+1:

**Comment by niranjan94** [comment]
+1

**Comment by jeffmelville** [comment]
Agreed, these indicators are better than anything else I have found. Have you considered BSD/MIT/LGPL?

**Comment by Jetroid** [comment]
:+1:

**Comment by JohanMabille** [comment]
👍

**Comment by cyboflash** [comment]
👍

**Comment by krystiangorecki** [comment]
:thumbsup:

**Comment by finnbear** [comment]
If anyone is interested, I drew a [100% original set of indicators](https://github.com/finnbear/yew_flight_indicators/tree/main/src/svg_part) and licensed them under MIT, Apache 2, or CC BY 4.0, at your option.

![example](https://github.com/user-attachments/assets/df7da92a-deb3-45eb-8982-d9acca0bc1ef)

However, they are **not a drop-in replacement**, as they have more features (inclinometer slip/skid, variable airspeed labels, 10,000 feet altitude warning, stall speed, warning speed, never exceed speed, autopilot heading) and different units (inHg, feet per minute, etc.).

Anyone is welcome to port them back to JQuery/JavaScript/etc. :heart:
